app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb

```diff
@@ -52,7 +54,9 @@ EOF
         subscription-manager clean
       fi
 
-      $PKG_MANAGER_REMOVE katello-ca-consumer\* > /dev/null 2>&1
+      if ![ -f /etc/debian_version ]; then
```

Comment:
What's the reason to skip this removal on Debian?

Reply:
There are no deb-packages generated for the katello-ca-consumer. Therefore, the `katello-ca-consumer`-script is used to install certificates and adapt subscription-manager config.
Which means there is no package installed that can be removed here.

Reply:
@lhellebr Thanks for reporting. I am surprised that we did not stumble across this before https://github.com/theforeman/foreman/pull/10161